src/axom/inlet/Inlet.cpp

```diff
@@ -255,27 +253,30 @@ bool Inlet::verifyValue(axom::sidre::Group* sidreGroup) {
       double val = sidreGroup->getView("value")->getScalar();;
       return checkRange(sidreGroup, val);
     }
+  } else if (sidreGroup->hasGroup("validStringValues")) {
```

Comment:
Can you add a specific check to make sure that a Field does not have both a validStringValues and validValues?  That is a case that shouldn't happen but we want to guard against.

Reply:
I added a warning for this in the validValues functions